### DISCUSSION TITLE
Changes large numbers when return the value

### DISCUSSION THREAD

**Discussion by GigantPro** [fastapi/fastapi#10151] [Questions] [answered]
### First Check

- [x] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI


app = FastAPI(
    title='test',
    openapi_url='/openapi.json',
    docs_url='/docs',
    root_path='/api',
)

@app.get('/', response_model=False)
async def test() -> int:
    return 814560837569871912
```

### Description

If you return large numbers (with or without nesting), but they somehow change their value

### Operating System

Linux

### Operating System Details

I am using Arch Linux and FireFox browser

### FastAPI Version

0.102.0

### Pydantic Version

2.3.0

### Python Version

Python 3.11.3

### Additional Context

I also ran a similar situation under docker (python:3.10.10-alpine3.17) and with FastAPI version 0.97.0
I work with the discord API and there such numbers act as an ID.
I tried to run without the response_model parameter - the same result.

**Comment by JavierSanchezCastro** [comment, marked as answer]
I think it's not FastAPI side, I've tried it with curl and Postman and it works fine.

I detect that is how the browser process and parse the data

![imagen](https://github.com/tiangolo/fastapi/assets/72013291/6ebf2331-e746-4bfa-8ea3-0506867d4ad3)

![imagen](https://github.com/tiangolo/fastapi/assets/72013291/56fecb3d-8390-4eea-835c-fc0a298adc04)

For the screenshots, I use this code: 

```python3
from fastapi import FastAPI
from pydantic import BaseModel

app = FastAPI()

class MyModel(BaseModel):
    a: int

@app.get('/')
async def test():
    a = 814560839999871912
    
    return MyModel(a=a)
```

Without the Model, the response is printed correctly in the network tab, but not in Swagger

**Reply by GigantPro** [reply]
Yes exactly! Thanks a lot for your answer and time!
